test/jdk/java/foreign/TestLayoutPaths.java

```diff
@@ -252,9 +214,9 @@ public void testStructPaths(IntFunction<PathElement> groupSelector) {
         long[] offsets = { 0, 8, 24, 56 };
         GroupLayout g = MemoryLayout.structLayout(
                 ValueLayout.JAVA_BYTE.withName("0"),
-                ValueLayout.JAVA_CHAR.withName("1"),
-                ValueLayout.JAVA_FLOAT.withName("2"),
-                ValueLayout.JAVA_LONG.withName("3")
+                ValueLayout.JAVA_CHAR.withBitAlignment(8).withName("1"),
+                ValueLayout.JAVA_FLOAT.withBitAlignment(8).withName("2"),
+                ValueLayout.JAVA_LONG.withBitAlignment(8).withName("3")
```

Comment:
Maybe there should be an alias for `withBitAlignment(8)`? E.g. `JAVA_LONG.withoutAlignment()`.